CONTRIBUTING.md

```diff
@@ -67,10 +67,30 @@ Make sure that commits are scoped to something meaningful and could potentially
 
 The body may contain a more detailed description of the commit, explaining what it changes and why. The "how" is less relevant, as this should be obvious from the diff.
 
+## Distributions
+
+The Things Stack docs cover all flavors of The Things Stack:
+
+- The Things Stack Cloud
+- The Things Stack Dedicated Cloud
+- The Things Stack Marketplace Launcher
+- The Things Stack Enterprise
+- The Things Stack Open Source
+- The Things Network
+
+Mark documentation that applies only to a specific distribution in one of the following ways:
+
+- Use the Front Matter `distributions` element to add a list of distributions, i.e `distribution: ["Enterprise", "Cloud"]`. This will mark the page in the parent's table of contents, and will produce a notification on the page
```

Comment:
But which are the valid distribution values?

Is "The Things Network" a distribution?

Reply:
OK, this question is answered below, please reorder